src/api/controllers/lines.js

```diff
@@ -283,7 +290,8 @@ function getMyPlaceByEmployerId(user_id, employer_id) {
 function getAverageWaitTimeByEmployerId(employer_id) {
     //get all line events for this employer from the past 24 hours.
     //sort them by user_id, then by time of event.
-    var yesterday = new Date() - MS_IN_ONE_DAY;
+    var yesterday = new Date();
```

Comment:
i think we should rename this variable to be more clear that it's midnight. rename to todaymidnight

Reply:
oh call it, beginningOfDay or something